test_runner/fixtures/utils.py

```diff
@@ -235,15 +235,18 @@ def allure_attach_from_dir(dir: Path):
             name = str(attachment.relative_to(dir))
 
             # compress files larger than 1Mb, they're hardly readable in a browser
-            if attachment.stat().st_size > 1024 * 1024:
-                source = f"{attachment}.tar.gz"
-                with tarfile.open(source, "w:gz") as tar:
-                    tar.add(attachment, arcname=attachment.name)
-                name = f"{name}.tar.gz"
+            if attachment.stat().st_size > 1024**2:
+                name = f"{name}.zst"
+                source = f"{attachment}.zst"
+                with open(source, "wb") as f:
+                    f.write(zstandard.compress(attachment.read_bytes()))
```

Comment:
Does this use multiple threads..?

Reply:
Docs don't say: https://python-zstandard.readthedocs.io/en/latest/misc_apis.html#compress

Reply:
There is a whole page about it: https://python-zstandard.readthedocs.io/en/latest/multithreaded.html

I think for all our runners have multiple cores, so it might be worth the amount of typing but might be that it's just my personal preferences and this will be a noticeable win even without it up to our usual attachment sizes.

Reply:
I tend to leave it as is, while we compress results in one pytest runner, others are still running, I don't want to introduce unexpected spikes in CPU usage just in case.

Also I believe that it's already fast enough